### PR TITLE
added a few links

### PR DESCRIPTION
**Pull request by idalithb**
One question was updated

### CODE REVIEW

File: website/pages/en/sunrise.mdx [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Sunrise of Decentralized Data FAQ
 ---
 
-> Note: This document is continually updated to ensure the most accurate and helpful information is provided. New questions and answers are added on a regular basis. If you can’t find the information you’re looking for, or if you require immediate assistance, [reach out on Discord](https://discord.gg/vtvv7FP).
+> Note: This document is continually updated to ensure the most accurate and helpful information is provided. New questions and answers are added on a regular basis. If you can’t find the information you’re looking for, or if you require immediate assistance, [reach out on Discord](https://discord.gg/vtvv7FP). If you are looking for billing information, then please refer to [billing](https://thegraph.com/docs/en/billing/).
```
[the remark this answers is not in the document]
Comment (benface) [reply website/pages/en/sunrise.mdx:5]: Oh, also, that Discord invite link is old. The correct/evergreen one is `https://discord.gg/graphprotocol`. So:
```suggestion
> Note: This document is continually updated to ensure the most accurate and helpful information is provided. New questions and answers are added on a regular basis. If you can’t find the information you’re looking for, or if you require immediate assistance, [reach out on Discord](https://discord.gg/graphprotocol). If you are looking for billing information, then please refer to [billing](/billing/).
```

File: website/pages/en/sunrise.mdx [hunk 2]
```diff
@@ -46,7 +46,7 @@ The Graph community is here to support developers as they move to The Graph Netw
 
 ### How can I ensure high quality of service and redundancy for subgraphs on The Graph Network?
 
-All subgraphs will be supported by the upgrade indexer. In order to attract additional Indexers for redundancy and improved quality of service, developers can add curation signal to subgraphs eligible for indexing rewards. [Learn more about adding signal to your subgraph](/publishing/publishing-a-subgraph/#adding-signal-to-your-subgraph).
+All subgraphs will be supported by the upgrade indexer. In order to attract additional Indexers for redundancy and improved quality of service, developers can add curation signal to subgraphs eligible for indexing rewards. [Learn more about adding signal to your subgraph and GRT](/publishing/publishing-a-subgraph/#adding-signal-to-your-subgraph).
```
Comment (benface) [inline website/pages/en/sunrise.mdx:49]: "adding signal to your subgraph and GRT"? I don't get the phrasing. 🤔

File: website/pages/en/sunrise.mdx [hunk 3]
```diff
@@ -82,7 +82,7 @@ Around the start of June, Edge & Node will automatically upgrade actively querie
 
 ### What happens if I don't upgrade my subgraph?
 
-Subgraphs will be queryable on the hosted service until June 12th, after which query endpoints will no longer be available and owners will not be able to deploy new versions. Owners will still be able to upgrade their subgraphs to The Graph Network at any time, even after June 12th.
+Subgraphs will be queryable on the hosted service until June 12th. After this date, the hosted service homepage will still be accessible, however, query endpoints will no longer be available to query. Owners of hosted service subgraphs will still be able to upgrade their subgraphs to The Graph Network after June 12th, though earlier upgrades are entitled to [earn rewards](https://staging.thegraph.com/sunrise-upgrade-program/). Developers will also be able to claim [auto-upgraded subgraphs](https://thegraph.com/blog/unveiling-updated-sunrise-decentralized-data/#phase-2-sunbeam)
```
Comment (benface) [inline website/pages/en/sunrise.mdx:85]:
```suggestion
Subgraphs will be queryable on the hosted service until June 12th. After this date, the hosted service homepage will still be accessible, however, query endpoints will no longer be available. Owners of hosted service subgraphs will still be able to upgrade their subgraphs to The Graph Network after June 12th, though earlier upgrades are entitled to [earn rewards](https://thegraph.com/sunrise-upgrade-program/). Developers will also be able to claim [auto-upgraded subgraphs](https://thegraph.com/blog/unveiling-updated-sunrise-decentralized-data/#phase-2-sunbeam)
```
Watch out for staging URLs. 😱

**Review by benface — CHANGES_REQUESTED** [review]
A couple things (check earlier comments).

File: website/pages/en/sunrise.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (idalithb) [reply website/pages/en/sunrise.mdx:5]: Great catch!

File: website/pages/en/sunrise.mdx (hunk 2 above)
Comment (benface) [reply website/pages/en/sunrise.mdx:49]: Existed before where? I'm confused. Looks like you added "and GRT".

Comment (idalithb) [inline website/pages/en/sunrise.mdx:49]:
```suggestion
All subgraphs will be supported by the upgrade indexer. In order to attract additional Indexers for redundancy and improved quality of service, developers can add curation signal to subgraphs eligible for indexing rewards. [Learn more about adding signal to your subgraph](/publishing/publishing-a-subgraph/#adding-signal-to-your-subgraph).
```
```suggestion
All subgraphs will be supported by the upgrade indexer. In order to attract additional Indexers for redundancy and improved quality of service, developers can add curation signal to subgraphs eligible for indexing rewards. [Learn more about adding signal to your subgraph and GRT](/publishing/publishing-a-subgraph/#adding-signal-to-your-subgraph).
```

**Review by benface — APPROVED** [review]
Looks good to me! Thank you.

File: website/pages/en/sunrise.mdx (hunk 2 above)
Comment (idalithb) [reply website/pages/en/sunrise.mdx]: Hi! Sorry, I confused it with another docs page phrasing.

Comment (benface) [reply website/pages/en/sunrise.mdx]: No worries :) I see you removed it.

**Pull request merged**